src/redux/missions/missions.js

```diff
@@ -0,0 +1,75 @@
+import missionData from '../../pages/missionsData';
+
+const FETCH_MISSION = 'spacex/missions/FETCH_MISSION';
+const JOIN_MISSION = 'spacex/missions/JOIN_MISSION';
+const LEAVE_MISSION = 'spacex/missions/LEAVE_MISSION';
+
+const initialState = [];
+
+// ACTIONS
+export const loadMissions = (payload) => ({
+  type: FETCH_MISSION,
+  payload,
+});
+
+export const joinMission = (id) => ({
+  type: JOIN_MISSION,
+  payload: {
+    id,
+  },
+});
+
+export const leaveMission = (id) => ({
+  type: LEAVE_MISSION,
+  payload: {
+    id,
+  },
+});
+
+const fetchMissions = () => async (dispatch) => {
+  // const response = await fetch(baseURL);
+  // const data = missionData
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove all commented code from your project as this is regarded as a bad practice.